autorest/sender.go

```diff
@@ -40,7 +40,7 @@ func (sf SenderFunc) Do(r *http.Request) (*http.Response, error) {
 	return sf(r)
 }
 
-// SendDecorator takes and possibily decorates, by wrapping, a Sender. Decorators may affect the
+// SendDecorator takes and possibility decorates, by wrapping, a Sender. Decorators may affect the
```

Comment:
I think this word is supposed to be `possibly`.

Reply:
Done.